Automattic/jetpack
# Protect: Add TypeScript Support

## Proposed changes:

* Adds `typescript` as a development dependency
* Adds `tsconfig.json`
  * Based off of existing projects' configurations, i.e. [Boost](https://github.com/Automattic/jetpack/blob/trunk/projects/plugins/boost/tsconfig.json)

### Other information:


## Jetpack product discussion



N/A

## Does this pull request change what data or activity we track or use?



No

## Testing instructions:






* Check out this branch
* Run `jetpack build plugins/protect`
* Ensure the plugin runs without regression.
